ui/src/app/components/search-input.tsx

```diff
@@ -89,7 +89,20 @@ export interface SearchInputProps {
   onChange: (newInput: string) => void;
 }
 
-export class SearchInput extends React.Component<SearchInputProps, SearchInputState> {
+export class SearchInput extends React.Component<any, SearchInputState> {
```

Comment:
Since SearchInputProps has been replaced by any, can the interface SearchInputProps be removed from search-input.tsx?

Reply:
My mistake -- I was trying to satisfy the Typescript compiler. I changed this back to use SearchInputProps.